TeamCode/src/main/java/org/firstinspires/ftc/teamcode/utilities/CONFIG.java

```diff
@@ -13,6 +13,10 @@ public class CONFIG {
     public static String BACK_LEFT = "bl";   // ch motor 3
     public static String BACK_RIGHT = "br";  // ch motor 1
 
+    //scissors
+    public static String scissorsServo = "scissors_servo";
```

Comment:
fix to match drivers hub config:
```java
public static String scissorsServo = "scissors"; // Not labeled, ch sp 2
public static String scissorsServoL = "scissorsL"; // Not labeled, eh sp 5
```

make sure to rename references and make sure the variable is intuitive ish